db/schema.rb

```diff
@@ -172,39 +172,39 @@
   SQL
 
   create_view :distributions,  sql_definition: <<-SQL
-      SELECT percentile_disc((ARRAY[0.99, 0.95, 0.90, 0.75, 0.50, 0.25])::double precision[]) WITHIN GROUP (ORDER BY date_part('epoch'::text, (vsc.created_at - v.created_at))) AS percentiles
+      SELECT percentile_disc(ARRAY[(0.99)::double precision, (0.95)::double precision, (0.90)::double precision, (0.75)::double precision, (0.50)::double precision, (0.25)::double precision]) WITHIN GROUP (ORDER BY date_part('epoch'::text, (vsc.created_at - v.created_at))) AS percentiles
```

Comment:
Oh, weird.  That must be a leftover from an old dev version of the DB that is still hanging around on this machine.   I'll fix it.